metadata-ingestion/src/datahub/ingestion/source/sql/sql_config.py

```diff
@@ -149,10 +145,7 @@ class SQLAlchemyConnectionConfig(ConfigModel):
         ),
     )
 
-    _database_alias_deprecation = pydantic_field_deprecated(
-        "database_alias",
-        message="database_alias is deprecated. Use platform_instance instead.",
-    )
+    _database_alias_deprecation = pydantic_removed_field("database_alias")
```

Comment:
in this and a few other places:
```suggestion
    _database_alias_removed = pydantic_removed_field("database_alias")
```